app/src/store/ThunkAlerts.re

```diff
@@ -0,0 +1,19 @@
+open AppStore;
+open StoreAlerts;
+open Source;
+
+let showAlert = (severity, message, dispatch, state, _source: source) => {
+  let alert = {severity, message, id: state.alerts.idCounter};
+
+  dispatch(AlertsAction(Show(alert)));
+
+  Js.Promise.make((~resolve, ~reject as _) =>
```

Comment:
We don't need this to be a promise, as their is not asynchronous interaction after the alert has been set.

Reply:
True, copy/pasta accident.